macos/FluentUI/Separator.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+
+@objc(MSFSeparatorOrientation)
+public enum SeparatorOrientation: Int {
```

Comment:
These are obvious, but since public they probably need documentation?

Reply:
Changed to: 

```swift
public enum SeparatorOrientation: Int {
    case horizontal // Separator with no intrinic width
    case vertical // Separator with no intrinic height
}
```